.gitignore

```diff
@@ -115,6 +115,7 @@ compile_commands.json
 pkg/ebpf/bytecode/build/**/*.d
 pkg/ebpf/bytecode/runtime/conntrack.go
 pkg/ebpf/bytecode/runtime/http.go
```

Comment:
Do we still need this?

Reply:
yes. after merging this change, the local instance might be committed accidentally to the main branch